modules/library/jdbc/src/test/java/org/geotools/jdbc/JDBCFeatureWriterOnlineTest.java

```diff
@@ -0,0 +1,52 @@
+/*
+ *    GeoTools - The Open Source Java GIS Toolkit
+ *    http://geotools.org
+ *
+ *    (C) 2019, Open Source Geospatial Foundation (OSGeo)
+ *
+ *    This library is free software; you can redistribute it and/or
+ *    modify it under the terms of the GNU Lesser General Public
+ *    License as published by the Free Software Foundation;
+ *    version 2.1 of the License.
+ *
+ *    This library is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *    Lesser General Public License for more details.
+ */
+package org.geotools.jdbc;
+
+import org.geotools.data.FeatureWriter;
+import org.geotools.data.Transaction;
+import org.opengis.feature.simple.SimpleFeature;
+
+/**
+ * JDBC {@link FeatureWriter}-implementation tampers with input-data.
+ *
+ * @author Burkhard Strauss
+ * @see GEOT-6264
+ */
+public abstract class JDBCFeatureWriterOnlineTest extends JDBCTestSupport {
+
+    public void testNext() throws Exception {
+
+        try (FeatureWriter writer =
```

Comment:
The description states "The test shows that attributes of features returned by JDBC-FeatureWriters are ok if retrieved via {{feature.getAttribute(index)}} but are {{null}} if retrieved via {{feature.getAttributes()}}."
But I don't see any call to getAttributes() here?

Reply:
Your're right my fault. The original test was to demonstrate the problem. The modified test must call getAttributes() and ensure they're not null. I will correct that.